report_presentation/Presenter.py

```diff
@@ -7,4 +16,274 @@ def __init__(self, report_root, initial_warnings, final_warnings, changes):
+        html1_7= '''
+                        <p class="serif" ><b>     1.  Number of Analysis Files:  %s</b></p>
+
+                        <p class="serif" ><b>     2.  Analysis Files:    </b></p>
+                        <p class="serif" >  %s </p>
+                        <p class="serif" ><b>     3.  Number of Warning Type:  %s </b></p>
+
+                        <p class="serif" ><b>     4.  Warning Code Type:    </b></p>
+                        <p class="serif" >  %s </p>
+                        <p class="serif" ><b>     5.  Total Number of Warnings: %s</b></p>
+                        <p class="serif" ><b>     6.  Distribution Graph: </b></p>
+                          <img src="images/before-overall.png" width="600" height="450" />
+                        <p class="serif" >
+                          <a href="subgraphs.html">7.   Sub Graphs for Every File</a>
+                        </p>
+                        ''' % (len(unique_warning_code), file_path_string, len(unique_file_path), warning_code_string, len(all_warnings))
+        return html1_7
+
+    def get_html8(self, all_warnings):
+        html = '''
+                        <p class="serif" ><b>     8.  Detailed Warning Information (Before CTA)</b></p>
+                        <div class = "before-warning-table">
+                        <table border = "0">
+                                <tr>
+                                        <th>No.</th>
+                                        <th>Warning Code</th>
+                                        <th>File Path</th>
+                                        <th>Project Name</th>
+                                        <th>Line Number</th>
+                                        <th>Warning message</th>
+                                </tr>'''
+        html += self.get_warning_info(all_warnings)
+        html += '''
+                        </table>
+                        </div>
+                        '''
+        return html
+
+    def get_html9(self, final_warnings):
+        html = '''
+                        <p class="serif" ><b>     9.  Detailed Warning Information (After CTA)</b></p>
+                        <div class = "after-warning-table">
+                        <table border = "0">
+                                <tr>
+                                        <th>No.</th>
+                                        <th>Warning Code</th>
+                                        <th>File Path</th>
+                                        <th>Project Name</th>
+                                        <th>Line Number</th>
+                                        <th>Warning message</th>
+                                </tr>'''
+        html += self.get_warning_info(final_warnings)
+        html += '''
+                        </table>
+                        </div>
+                        '''
+        return html
+
+    def get_html10(self, changes):
```

Comment:
here and above, what do these numbers stand for? I think using more descriptive identifier names would improve readablity